helpers/guilds/automod/createAutomodRule.ts

```diff
@@ -61,6 +61,8 @@ export interface CreateAutoModerationRuleOptions {
     presets?: DiscordAutoModerationRuleTriggerMetadataPresets[];
     /** The substrings which will exempt from triggering the preset trigger type. Only present when TriggerType.KeywordPreset */
     allowList?: string[];
+    /** Total number of mentions (role & user) allowed per message (Maximum of 50) */
```

Comment:
i imagine this also needs to be fixed above in the code that sends this to discord